flake.nix

```diff
@@ -17,31 +17,41 @@
       ];
       perSystem =
         {
+          system,
           self',
           pkgs,
+          pkgsEulaAccepted,
           ...
         }:
         {
-          formatter = pkgs.nixfmt-rfc-style;
-
-          packages = {
-            premenv = pkgs.callPackage ./premenv.nix { };
```

Comment:
Deleted because we wouldn't want anyone to be unable to accept the EULA now, would we. (More specifically, with the pkgs in this closure, it is now actually impossible to accept the EULA because we don't overlay it in _module.args).